config/config.go

```diff
@@ -240,45 +240,47 @@ type EnvironmentConfig struct {
 	// Unique ID of the environment config
 	ID string `yaml:"id" json:"id"`
 
-	// A list of proxy configs
-	ProxyConfigs []ProxyConfig `yaml:"proxy_configs" json:"proxy_configs"`
+	// A list of API configs.
+	APIs []APIConfig `yaml:"apis" json:"apis"`
 }
 
-// ProxyConfig has the proxy configuration.
-type ProxyConfig struct {
-	// Top-level basepath for the proxy config
-	Basepath string `yaml:"basepath,omitempty" json:"basepath,omitempty"`
+// APIConfig contains settings for an individual API.
+type APIConfig struct {
+	// ID of the API, used to match the api_source of API Product Operations.
+	ID string `yaml:"id" json:"id"`
+	
+	// Base path for this API.
+	BasePath string `yaml:"base_path,omitempty" json:"base_path,omitempty"`
 
-	// Authentication defines the proxy-level authentication requirement
+	// The default authentication requirements for this API.
 	Authentication AuthenticationRequirement `yaml:"authentication,omitempty" json:"authentication,omitempty"`
 
-	// ConsumerAuthorization defines the proxy-level consumer authorization
+	// The default consumer authorization requirements for this API.
 	ConsumerAuthorization ConsumerAuthorization `yaml:"consumer_authorization,omitempty" json:"consumer_authorization,omitempty"`
 
-	// Name of the target server for this proxy.
-	Target string `yaml:"target" json:"target"`
+	// The default value for the `x-apigee-target` header that will be appended to all allowed requests. 
+	TargetID string `yaml:"target" json:"target"`
 
-	// A list of Operations, names of which must be unique within the proxy config.
+	// A list of API Operations, names of which must be unique within the API.
```

Comment:
I missed this in my PR. Do we allow an API to have no operations? What would happen if so?

Reply:
We need at least one, kinda like having at least one API Proxy per Environment.

Reply:
Makes sense. So we need to remove the `omitempty` tag.